packages/clerk-js/src/ui/elements/AvatarUploader.tsx

```diff
@@ -101,7 +102,7 @@ export const AvatarUploader = (props: AvatarUploaderProps) => {
             )}
           </Flex>
           <Text
-            sx={t => ({ color: t.colors.$blackAlpha400 })}
+            sx={t => ({ color: t.colors.$blackAlpha400, fontSize: t.fontSizes.$sm })}
```

Comment:
Can't we use a predefined text variant instead of the `fontSize` directly?

Reply:
The only current variant that uses the `sm` size we have is `buttonSmall`, which wouldn't work here.
Should I create a new variant for it then?

Reply:
@marcelscruz You are right. This is wrong on the designs. They should be using either the `caption` or the `body` text. You can ask the design team about this. We should stick to the existing typography.

Reply:
Waiting for their feedback on Figma to fix this.